tests/commits/check.sh

```diff
@@ -37,9 +37,14 @@ git log --no-merges --pretty="%H" $commit_range -- | (
   while read commit; do
     commit_has_valid_subject=1
     commit_has_signed_off_by=1
+    revert_commit=0
 
     subject=$(git show -s --format='%s' $commit)
 
+    if echo "$subject" | egrep -q '^Revert '; then
+      revert_commit=1
+    fi
```

Comment:
Do we really want the "revert" commits to be excluded ?
The revert commit does not really have any details that distinguish it from other commits (message can be altered any point).
By including the `Signed-off` in the commit message, does not really mean anything at all. (git at least does not specify, and I could not find any statement in the syslog-ng repository either, except that you must include in every commit)

Despite the above I like `Signed-Off` as you can easily give credit with it to other people.

Reply:
We have 22 revert commits in this repository, only 3 of them contain signoff lines. We can start using signoffs there too.

Reply:
thanks.